velox/dwio/common/tests/utils/E2EFilterTestBase.h

```diff
@@ -219,7 +219,7 @@ class E2EFilterTestBase : public testing::Test {
       bool forRowGroupSkip) = 0;
 
   virtual std::unique_ptr<dwio::common::Reader> makeReader(
-      const dwio::common::ReaderOptions& opts,
+      dwio::common::ReaderOptions& opts,
```

Comment:
This is not clean, can you add a `virtual void setUpReaderOptions(ReaderOptions&) {}` and call it before `makeReader` instead?  Then only override that in DWRF subclass.

Reply:
I have added the corresponding method